pymc/data.py

```diff
@@ -22,6 +22,7 @@
 
 import numpy as np
 import pandas as pd
+import polars as pl
```

Comment:
This should be optional dependency (pandas also but that is futile because arviz requires it)